test/e2e.integration.test.js

```diff
@@ -22,29 +22,31 @@ import {
   uint8tohex,
   braveYoutubeOwner,
   braveYoutubePublisher,
-  cleanRedisDb
+  cleanRedisDb,
+  numBatchVotes,
+  voteExchangeRate,
+  suggestedVotes
 } from './utils'
 
 import dotenv from 'dotenv'
 dotenv.config()
 
-const voteExchangeRate = 1 // 1 BAT per vote
-const suggestedVotes = 12
-const numBatchVotes = 2
-
 let prevSurveyorId
 
 test('ledger: create a surveyor', async t => {
   // need access to eyeshade db
   t.plan(3)
   const url = '/v2/surveyor/contribution'
-  const data = { 'adFree': {
-    'fee': { 'USD': 5 },
-    'votes': suggestedVotes,
-    'altcurrency': 'BAT',
-    'probi': new BigNumber(suggestedVotes * voteExchangeRate).times('1e18').toString()
-  }}
-
+  const data = {
```

Comment:
should we use the new createSurveyor function?

Reply:
data and url are used in subsequent lines

Reply:
I think we can just call createSurveyor twice.
```
createSurveyor()
id1 =     .get('/v2/surveyor/contribution/current')
createSurveyor()
id2 =     .get('/v2/surveyor/contribution/current')